.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: node_js
 node_js:
+  - '8'
+  - '7'
   - '6'
   - '5'
   - '4'
-  - '0.12'
-  - '0.10'
 git:
   submodules: false
+before_install: npm install -g npm
```

Comment:
There was a reason for this - it could be due to an existing dependency not installing correctly on npm 2.